src/java/org/apache/cassandra/tools/nodetool/Info.java

```diff
@@ -59,14 +59,29 @@ public void execute(NodeProbe probe)
             out.printf("%-23s: %s%n", "Generation No", 0);
 
         // Uptime
-        long secondsUp = probe.getUptime() / 1000;
-        out.printf("%-23s: %d%n", "Uptime (seconds)", secondsUp);
+        try
```

Comment:
these try blocks are here because when tested with JMX feature, Nodetool will not find Memory bean on Instance (weird, right?), same for heap memory usage. Basically when it tries to connect to JMX and it initialises mbeans, it will not find Memory and the one below.